packages/react-components/react-popover/src/components/PopoverSurface/usePopoverSurface.ts

```diff
@@ -49,6 +49,7 @@ export const usePopoverSurface_unstable = (
       role: trapFocus ? 'dialog' : 'group',
       'aria-modal': trapFocus ? true : undefined,
       ...modalAttributes,
+      tabIndex: '-1',
```

Comment:
in cases where popovers don't contain focusable elemnts, wouldn't users just be able to use `inline`?

There's nothing in this component that will actually focus this element if it's open. Unless the conclusion behind this PR is that we should be doing that?

Reply:
@smhigley probably has more insight to share

Reply:
@ling1726 the popover does actually get focus when `tabIndex="-1"` is present -- my assumption was that tabster was doing that 😅

Reply:
PopoverSurface gets focus when `tabIndex=-1` because of this line 

https://github.com/microsoft/fluentui/blob/517b807cd97ee89a470ac5e9dec48a6f42b69532/packages/react-components/react-popover/src/components/Popover/usePopover.ts#L129

Reply:
When the popover was built, we never expected the focusing effect to work when users don't put focusable elements inside popovers. Our guidance has always been to treat popovers as dialogs when this happens use `trapFocus` in that case.

@smhigley  if popovers have no focusable content, would `inline` be enough? if we result to focusing the entire popover by default always, I don't  see a reason to use `inline`